pkg/quarterdeck/db/models/orgs.go

```diff
@@ -315,6 +324,14 @@ func (o *Organization) ProjectCount() int {
 	return o.projects
 }
 
+// Return the last login timestamp for the user this organization was loaded for.
+func (o *Organization) LastLogin() (time.Time, error) {
+	if o.lastLogin.Valid {
+		return time.Parse(time.RFC3339Nano, o.lastLogin.String)
+	}
+	return time.Time{}, nil
```

Comment:
Just wanted to note that this logic requires that no login is NULL and not empty string. The `DEFALT NULL` on the schema means that this will start invalid but once set should always be set (which is the correct logic).